include/picongpu/fields/differentiation/Traits.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2013-2022 Heiko Burau, Rene Widera, Axel Huebl, Sergei Bastrakov
```

Comment:
Same here. May name can be removed to because I have not contributed to the file.